src/main/target/REVONANO/target.h

```diff
@@ -74,6 +74,7 @@
 
 #define USE_I2C
 #define USE_I2C_DEVICE_1        // UART1/FlexiPort (PB6,PB7)
+#define USE_I2C_DEVICE_2        // UART1/FlexiPort (PB6,PB7)
```

Comment:
The comment may be wrong

Reply:
Ah, I did this for testing. Will remove it.